packages/devtools/test/inspector_controller_test.dart

```diff
@@ -544,5 +544,75 @@ void main() async {
 
       await env.tearDownEnvironment();
     });
-  }, tags: 'useFlutterSdk', timeout: const Timeout.factor(2));
+
+    test('hotRestart', () async {
+      await env.setupEnvironment();
+
+      // Select row index 4.
+      // The important thing about this is that the details tree should scroll
+      // instead of re-rooting as the selected row is already visible in the
+      // details tree.
+      tree.onTap(const Offset(0, rowHeight * 4.5));
```

Comment:
probably me just not being super familiar with this test, but where does 4.5 come from?

Reply:
just in the middle of row index 4.

Reply:
added a 
tapRow helper to make that self documenting.

Reply:
correction. clickRow